exercises/concept/vec-macro/src/lib.rs

```diff
@@ -1,9 +1,5 @@
-// This stub file contains items which aren't used yet; feel free to remove this module attribute
-// to enable stricter warnings.
```

Comment:
IMO we should leave this as documentation for the student.

Reply:
Ah, I see what you mean. Yes, I'll leave it itn.